api/tests/opentrons/protocol_engine/commands/temperature_module/test_deactivate.py

```diff
@@ -43,4 +43,4 @@ async def test_await_temperature(
 
     result = await subject.execute(data)
     decoy.verify(await tempdeck_hardware.deactivate(), times=1)
-    assert result == temperature_module.AwaitTemperatureResult()
+    assert result == temperature_module.WaitForTemperatureResult()
```

Comment:
Oh boy, this is my bad. I guess the test passed since both deactivate and awaitTemperature results were empty objects. Let's improve that test:
```suggestion
    assert result == temperature_module.DeactivateTemperatureResult()
    assert isinstance(result, temperature_module.DeactivateTemperatureResult)
```